bitnami/redis-cluster/templates/update-cluster.yaml

```diff
@@ -103,7 +105,7 @@ spec:
                   sleep 5
                 done
                 slave=()
-                if (( $REDIS_CLUSTER_REPLICAS >= 1 )) && (( newNodeCounter % (( $REDIS_CLUSTER_REPLICAS + 1 )) )); then
+                if (( $REDIS_CLUSTER_REPLICAS >= 1 )) && (( (( $newNodeCounter < $slaveNodesEndPos )) || (( (( $newNodeCounter >= $slaveNodesEndPos )) && (( $newNodeCounter % (( $REDIS_CLUSTER_REPLICAS + 1 )) )) )) )); then
```

Comment:
I would clarify this condition with a comment since it is hard to read

Reply:
Done.Please review it again.Thank you^_^